c7n/resources/s3.py

```diff
@@ -332,6 +332,7 @@ def handle_BucketReplicationConfiguration(self, resource, item_value):
                 'Status': r['status'],
                 'Prefix': r['prefix'],
                 'Destination': {
+                    'Account': r['destinationConfig']['Account'],
```

Comment:
oh cool, didn't even realize this was available, and it looks like its in the describe output as well.